### DISCUSSION TITLE
want to load data using the URL load method after the initialisation by calling the store manager API,

### DISCUSSION THREAD

**Discussion by swapnart** [GrapesJS/grapesjs#3842] [Q&A]
Hi, since so far, i am using your plugin and its works nice but i have one question i want to load the data dynamically to the editor , that based on the editor id but i have initiated the editor but later on automatically i have to load data based on some logic where url load does not allowed me to build the logic,basically i want to initiate the editor first and load the data from the remote api call using the storemanager api

**Comment by artf** [comment]
You can use [`editor.load`](https://grapesjs.com/docs/api/editor.html#load)
